src/Microsoft.DotNet.Wpf/src/Shared/System/Windows/Markup/ReflectionHelper.cs

```diff
@@ -78,65 +78,48 @@ internal static Type GetQualifiedType(string typeName)
         {
             // ISSUE: we only parse the assembly name and type name
             // all other Type.GetType() type fragments (version, culture info, pub key token etc) are ignored!!!
-            string[] nameFrags = typeName.Split(new Char[] { ',' }, 2);
-            Type type = null;
+            string[] nameFrags = typeName.Split(new char[] { ',' }, 2);
             if (nameFrags.Length == 1)
             {
                 // treat it as an absolute name
-                type = Type.GetType(nameFrags[0]);
+                return Type.GetType(nameFrags[0]);
             }
             else
```

Comment:
Since you added a `return` above, you could remove this `else` to reduce nesting